tests/integration-all/fileSystemConfig/service/serverless.yml

```diff
@@ -0,0 +1,12 @@
+service: CHANGE_TO_UNIQUE_PER_RUN
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  versionFunctions: false
+
+functions:
+  writer:
+    handler: core.writer
+  reader:
+    handler: core.reader
```

Comment:
Let's maybe add comment, that in integration tests VPC and EFS configuration is added dynamically right before test is run.